src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -154,6 +154,15 @@ NRQL's basic rules are as follows:
         NRQL does not have the equivalent of the SQL `JOIN` clause, but you can [simulate a `JOIN` with custom attributes](/docs/insights/new-relic-insights/using-new-relic-query-language/simulate-sql-join-functions-insights).
       </td>
     </tr>
+    <tr>
+      <td>
+        Comments
+      </td>
+
+      <td>
+        You can add non-working comments to your NRQL queries. [Learn more about comments](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions#comments). 
```

Comment:
I think "non-working" sounds strange here, I would omit.  Other than that, this looks good!  Thank you!